lnd.go

```diff
@@ -675,10 +675,30 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 
 	// With all the relevant chains initialized, we can finally start the
 	// server itself.
-	if err := server.Start(); err != nil {
+	errChan := make(chan error)
```

Comment:
nit: maybe a good idea just to add a comment about this new async start style. I think it isnt super clear from the context why we do it this way now